prow/pubsub/subscriber/subscriber.go

```diff
@@ -415,6 +425,10 @@ func (s *Subscriber) handleProwJob(l *logrus.Entry, jh jobHandler, msg messageIn
 		reportProwJobFailure(&prowJob, err)
 		return err
 	}
-	l.Infof("Job %q created as %q", pe.Name, prowJob.Name)
+	l.WithFields(logrus.Fields{
+		"event-name": pe.Name,
+		"job-name":   prowJob.Name,
```

Comment:
IIRC these should be `job` and `name` respectively to be consistent with how we filter on these elsewhere.

Reply:
fixed